.circleci/config.yml

```diff
@@ -36,59 +41,61 @@ jobs:
     executor:
       name: node/default
       tag: '10.16'
-    environment:
-      EXCHANGE_DOWNLOAD: true
     steps:
-      - run: echo Publish commerce_sdk to npm "$EXCHANGE_DOWNLOAD"
       - checkout
-        # Add read/write ssh key's finger print in order to push the tags
-      - add_ssh_keys:
-          fingerprints:
-            - "78:6d:b4:67:eb:35:d4:a0:63:af:c2:32:f1:94:9a:5f"
       - node/with-cache:
           steps:
             - run: npm install
       - run: npm run build
-      - run: echo -e "$GPG_KEY" | gpg --import
-      - run: git config --global user.email "$GITHUB_PUBLISH_EMAIL"
-      - run: git config --global user.name "$GITHUB_PUBLISH_USER"
-      - run: git config --global user.signingkey $GITHUB_GPG_SIGN_KEY_ID
-      - run: git config --global commit.gpgsign true
       - run:
           name: Create npmrc file to authenticate with npm registry.
           command: echo "//registry.npmjs.org/:_authToken=$NPM_AUTH_TOKEN" > ~/.npmrc
       - run: npm run publish
-      - run: sudo npm install -g @octokit/rest
-      - run: node .circleci/create_pr.js
-
 workflows:
   version: 2
   build-and-test:
     jobs:
       - build:
           filters:
-            branches:
-              ignore:
-                - /^release.*/
+            tags:
+              ignore: /^v.*/
       - build-12:
-        filters:
-          branches:
-            ignore:
-              - /^release.*/
-  build-test-and-deploy:
+          filters:
+            tags:
+              ignore: /^v.*/
+  build-test-deploy:
     jobs:
       - build:
           filters:
+            tags:
+              only: /^v.*/
+            branches:
+              ignore: /.*/
+      - build-12:
+          filters:
+            tags:
+              only: /^v.*/
+            branches:
+              ignore: /.*/
+      - confirm-publish-if-tagged:
+          filters:
```

Comment:
duplicate filters can be removed for approval step, require build and build12 will take care of filtering

Reply:
I had issues with this.  I initially didn't have it but then it wasn't running the step for some reason.  I didn't want to play iwth it much more as this worked and I did find something If you look at https://circleci.com/docs/2.0/workflows/#executing-workflows-for-a-git-tag there is an example where they have the filter at each step so I just used their example for this.

Reply:
Ok, Removing the filters from subsequent steps should work, 

not a big deal anyways. Just the config will be more cleaner..